plugins/catalog-common/api-report.md

```diff
@@ -0,0 +1,37 @@
+## API Report File for "@backstage/plugin-catalog-common"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { Permission } from '@backstage/plugin-permission-common';
+
+// @public
+export const catalogEntityReadPermission: Permission;
+
+// @public
+export const catalogEntityRefreshPermission: Permission;
+
+// @public
+export const catalogEntityUnregisterPermission: Permission;
+
+// @public
+export const catalogLocationCreatePermission: Permission;
+
+// @public
+export const catalogLocationDeletePermission: Permission;
+
+// @public
+export const catalogLocationReadPermission: Permission;
+
+// Warning: (ae-missing-release-tag) "RESOURCE_TYPE_CATALOG_ENTITY" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export const RESOURCE_TYPE_CATALOG_ENTITY = 'catalog-entity';
+
+// Warning: (ae-missing-release-tag) "RESOURCE_TYPE_CATALOG_LOCATION" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export const RESOURCE_TYPE_CATALOG_LOCATION = 'catalog-location';
+
+// (No @packageDocumentation comment for this package)
```

Comment:
Could we make it so that this one is free of warnings and `(undocumented)`? I think that's a good goal going forward overall.

Totally fine to do that in a later PR though.